open-api/rest-catalog-open-api.yaml

```diff
@@ -3952,121 +4018,102 @@ components:
             type: integer
           description: "List of equality field IDs"
 
-    PlanTableRequest:
+    PlanTableScanRequest:
       type: object
       properties:
         snapshot-id:
           description:
-            The ID of the snapshot to use for the table scan.
+            Identifier for the snapshot to scan in a point-in-time scan
           type: integer
           format: int64
         select:
-          description:
-            A list of fields in schema that are selected in a table scan.
-            When not specified, all columns in the requested schema should be selected.
+          description: List of selected schema fields
           type: array
           items:
             $ref: '#/components/schemas/FieldName'
         filter:
           description:
-            an unbounded expression to describe the filters to apply to a table scan,
+            Expression used to filter the table data
           $ref: '#/components/schemas/Expression'
         case-sensitive:
-          description: If field selection and filtering should be case sensitive
+          description: Enables case sensitive field matching for filter and select
           type: boolean
           default: true
         use-snapshot-schema:
           description:
-            If the client is performing time travel, the snapshot schema should be used.
-            For clients performing a plan for a branch, should default to using the table schema.
+            Whether to use the schema at the time the snapshot was written.
+
+            When time travelling, the snapshot schema should be used (true).
+            When scanning a branch, the table schema should be used (false).
           type: boolean
           default: false
         start-snapshot-id:
-          description: The ID of the starting snapshot of the incremental scan
+          description: Starting snapshot ID for an incremental scan (exclusive)
```

Comment:
I updated this to note that the starting ID is always exclusive and the ending is always inclusive.